spec.md

```diff
@@ -134,11 +134,25 @@ When restricting the issuer MUST add another path segment to the resource URI. U
 
 ```
 
-## 4.2 Capabilities
+## 4.2 Actions
```

Comment:
Do we need an additional action for `/LISTING`?

Reply:
we will but that required db schema changes, it will be the next phase of the ucan implementation